src/site/content/en/blog/fast-ads-matter/index.md

```diff
@@ -0,0 +1,196 @@
+be a drag on user experience metrics like [session
+length](https://en.wikipedia.org/wiki/Session_(web_analytics)) and [bounce
+rate](https://en.wikipedia.org/wiki/Bounce_rate). You can improve these metrics
+by serving the most lightweight ads possible and loading them at the right time
+(which is not always right away).
+
+For many e-commerce publishers, display ads are a secondary source of revenue.
+If you're one of these publishers, you know that any ads you place on the page
+have some negative impact on your primary business metrics (sales,
+subscriptions, and more). Fast ads, by getting out of the page's way, give your
+primary business metrics a boost as well.
+
+{% Aside %} When asked about their reasons for installing ad blockers, [many
+users cited "interruption" and
+"speed"](https://pagefair.com/blog/2017/adblockreport/) as primary motivators.
+Since fast ads result in improved user experience metrics, a focus on improving
+ad speed may decrease the incentive for users to install ad blockers. {%
+endAside %}
+
+### Fast ads make you more money
+
+Another way to think about this topic is from an advertiser's point of view. The
+sooner an ad appears on the page, the longer it will be visible on the screen,
+meaning it's more likely to be seen and interacted with. As views and
+interactions increase, so does the value of your ad slots in the eyes of
+advertisers.
+
+Conversely,
+[impressions](https://en.wikipedia.org/wiki/Impression_(online_media)) and
+[viewable impressions](https://en.wikipedia.org/wiki/Viewable_Impression)
+decrease the longer an ad takes to appear on the page. To provide a sense of the
+magnitude of this problem, the charts below show aggregated data from an
+experiment where a delay between 100&nbsp;ms and 1&nbsp;s was injected before
+each ad response, across 4 billion impressions on websites with the Google
+Publisher Tag in multi-request mode. The dotted lines are extrapolations to
+visualize how improving ad speed could increase impressions and viewability
+rate.
+
+With 1&nbsp;s of added delay, impressions decreased by 1.1% for mobile traffic
+and 1.9% for desktop traffic:
+
+<figure class="w-figure w-figure--center">
+  <img src="./ad-latency-injected-vs-impressions-change.svg" alt="Chart showing latency injected vs. impressions change">
+  <figcaption class="w-figcaption">
+    Source: Google Internal Data, December 2016 to January 2017.
+  </figcaption>
+</figure>
+
+With 1&nbsp;s of added delay, viewability rate decreased by 3.6% for mobile
+traffic and 2.9% for desktop traffic:
+
+<figure class="w-figure w-figure--center">
+  <img src="./ad-latency-injected-vs-viewability-rate-change.svg" alt="Chart showing latency injected vs. viewability rate change">
+  <figcaption class="w-figcaption">
+    Source: Google Internal Data, December 2016 to January 2017.
+  </figcaption>
+</figure>
+
+## A framework for thinking about ad speed
```

Comment:
FYI @mihajlija this post is more than 1000 words, which is one of the things the [checklist](https://web.dev/handbook/content-checklist/#basics) asks us to avoid. I think we could have split this post up into 2 post. The second one would have started here. The first post tells you why you should care about ad speed. The second post tells you how to approach the problem.

There's no action to take. I'm OK with shipping this slightly-long post. Maybe it's better as it is. I haven't thought about it in-depth. Just wanted to put this idea in your head for future reference.